src/app/components/CookieConsent/index.tsx

```diff
@@ -0,0 +1,116 @@
+import { useState } from 'react'
+import Snackbar from '@mui/material/Snackbar'
+import Typography from '@mui/material/Typography'
+import { styled } from '@mui/material/styles'
+import Box from '@mui/material/Box'
+import { COLORS } from '../../../styles/theme/colors'
+
+const CookieConsent = () => {
+  const [open, setOpen] = useState(true)
+
+  const handleClose = () => {
+    setOpen(false)
+  }
+
+  const AcceptCookiesButton = styled(Box)(({ theme }) => ({
+    width: '100%',
+    height: '47px',
+    borderRadius: '46px',
+    backgroundColor: COLORS.brandMedium,
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    fontSize: '18px',
+    color: 'white',
+    fontWeight: '400',
+    marginRight: '0',
+    marginBottom: '16px',
+    [theme.breakpoints.up('sm')]: {
+      width: '126px',
+      marginRight: '8px',
+      marginBottom: '0',
+    },
+  }))
+
+  const DeclineCookiesButton = styled(Box)(({ theme }) => ({
+    width: '100%',
+    height: '47px',
+    borderRadius: '46px',
+    backgroundColor: COLORS.white,
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    fontSize: '18px',
+    color: COLORS.graphLabel,
+    fontWeight: '400',
+    borderStyle: 'solid',
+    borderWidth: '1px',
+    borderColor: COLORS.graphLabel,
+    [theme.breakpoints.up('sm')]: {
+      width: '126px',
+      marginLeft: '8px',
+    },
+  }))
+
+  return (
+    <Snackbar
```

Comment:
Hm, maybe snackbar isn't optimal. Mobile users can't ignore and scroll past it

![localhost_1234_mainnet_emerald_token_0x903d32d7307b4b3FC4bc9a510CA658C91A346A67(extension360)](https://github.com/oasisprotocol/explorer/assets/3758846/09d610e5-b78d-4cca-8cf6-3de39e98bba4)